cpp/ql/test/README.md

```diff
@@ -0,0 +1,33 @@
+# C/C++ CodeQL tests
+
+This document provides additional information about the C/C++ CodeQL Tests located in `cpp/ql/test`.  See [Contributing to CodeQL](CONTRIBUTING.md) for general information about contributing to this repository.
+
+The tests can be run through Visual Studio Code.  Advanced users may also use the `odasa qltest` command.
```

Comment:
Surely this should be `codeql test run`?

Reply:
Thank you.  I was suspicious it had been replaced.

Reply:
Fixed.